test/unit/components/compose-comment.test.jsx

```diff
@@ -555,20 +555,46 @@ describe('Compose Comment test', () => {
         global.Date.now = realDateNow;
     });
 
-    test('getMuteMessageInfo: muteType set', () => {
+    test('getMuteMessageInfo: muteType set and just got muted', () => {
         const commentInstance = getComposeCommentWrapper({}).instance();
         commentInstance.setState({muteType: 'unconstructive'});
-        expect(commentInstance.getMuteMessageInfo().commentType).toBe('comment.type.unconstructive');
+        expect(commentInstance.getMuteMessageInfo(true).commentType).toBe('comment.type.unconstructive');
```

Comment:
Super minor, and not necessary to land. I just realized that it would be a little more self documenting if it declared a const for the boolean:
```suggestion
        const justMuted = true;
        expect(commentInstance.getMuteMessageInfo(justMuted).commentType).toBe('comment.type.unconstructive');
```